app/components/schools/teacher_profile_summary_list_component.html.erb

```diff
@@ -38,7 +38,7 @@
 
   <% list.with_row do |row| %>
     <% row.with_key { "Status" } %>
-    <% row.with_value { ect_status_tag } %>
+    <% row.with_value { ect_status(@ect) } %>
   <% end %>
```

Comment:
Just a thought - are we going to be adding the ability to change this status to be consistent with the rest of the change details view? Not on this ticket I am sure...